js/hoverzoom.js

```diff
@@ -1083,43 +1083,84 @@ var hoverZoom = {
             }
         }
 
-        let longRightPressTimer; // create timer
-        let longMiddlePressTimer; // creates separate timer so they don't interfere
-        let longRightPress = false;
+        // create right and middle button timers to handle them separately
+        let longRightPressTimer;
+        let longMiddlePressTimer; 
+        // TODO: Prevent middle click from triggering when it's an action key
```

Comment:
Is this still TODO or is this comment a leftover from the early development?

Reply:
It's something that is todo, but I couldn't figure it out yet